rivery_cli/rivery_session.py

```diff
@@ -366,7 +367,15 @@ def fetch_run_logs(self, **kwargs):
         url = '/activities/runs/logs'
         method = 'get'
         param = {"id": run_id}
-        return self.handle_request(url=url, method=method, params=param, return_full_response=return_full_response)
+
+        # Add query id if exists
+        query_id = kwargs.get('query_id')
+        if query_id:
```

Comment:
if you didn't get query_id, it will ask for new log again and again?

Reply:
fixed (threw exception in case query id is not provided, it shouldn't happen but it's definitely more correct code-wise to do so)